api/cards/updateCards.spec.ts

```diff
@@ -0,0 +1,31 @@
+import * as httpMocks from "node-mocks-http";
+import "../../utils/cards";
```

Comment:
Maybe it's useful to make absolute imports when this app is not so big. I have found solution with help of `.env` file with `NODE_PATH=src/` but I don't know if it generally a good idea.
Any thoughts ?

Reply:
I've added `paths` to ts configuration to use `~/utils/cards`